monitoring-dashboard/integration/org.wso2.micro.integrator.dashboard.integration.tests/src/test/resources/automation.xml

```diff
@@ -58,7 +58,7 @@
     Database configuration to be used for data service testing. DB configuration in dbs files will be replaced with
                below configuration at test run time
     -->
-    <datasources>
+    <!--datasources>
```

Comment:
What is the reason for commenting this?

Reply:
commented for testing initially, will remove it